### PR TITLE
chore: deployment to arbitrum sepolia

### PR DESCRIPTION
**Pull request by ghost**
## Description

Updated addresses for the recent deployment to Arbitrum Sepolia.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

-   [X] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
-   [ ] ~added `!` to the type prefix if API or client breaking change~
-   [ ] ~provided a link to the relevant issue or specification~
-   [ ] ~included the necessary unit and integration tests~
-   [ ] ~code is adequately commented~
-   [ ] ~updated the relevant documentation or specification~
-   [X] reviewed "Files changed" and left comments if necessary
-   [x] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

-   [x] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
-   [ ] ~confirmed `!` in the type prefix if API or client breaking change~
-   [ ] ~confirmed all author checklist items have been addressed~
-   [ ] ~reviewed service design and naming~
-   [ ] ~reviewed documentation is accurate~
-   [ ] ~reviewed tests and test coverage~
-   [ ] ~manually tested (if applicable)~

### CODE REVIEW

**Comment by ghost** [comment]
@ElliotFriedman @utx0 - will need to cleanup the integration tests so that we can correctly test against the Arbitrum mainnet fork, but for this PR I've updated the tests to fork Goerli instead. Can address this in a subsequent PR.

File: script/utils/nfts/mintToken.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    .requiredOption("-a, --units <amount>", "Number of NFTs to mint")
+    .requiredOption(
+        "-u, --rpc-url <url>",
+        "RPC URL",
+        process.env.TESTNET_RPC_URL || ""
+    );
+
+program.parse();
+
+const abi = JSON.parse(fs.readFileSync(program.opts().abiPath, "utf-8"));
```
Comment (ElliotFriedman) [inline script/utils/nfts/mintToken.ts:32]: if you're always going to call the admin minter, you should be able to hardcode the path to the abi, that way you don't have to pass it when using.

**Review by ElliotFriedman — APPROVED** [review]

**Comment by utx0** [comment]
> @ElliotFriedman @utx0 - will need to cleanup the integration tests so that we can correctly test against the Arbitrum mainnet fork, but for this PR I've updated the tests to fork Goerli instead. Can address this in a subsequent PR.

Sorry why did you switch it away from `ARBITRUM_TESTNET_SEPOLIA_RPC_URL` not sure I understand your point here?

**Comment by ghost** [comment]
> > @ElliotFriedman @utx0 - will need to cleanup the integration tests so that we can correctly test against the Arbitrum mainnet fork, but for this PR I've updated the tests to fork Goerli instead. Can address this in a subsequent PR.
> 
> Sorry why did you switch it away from `ARBITRUM_TESTNET_SEPOLIA_RPC_URL` not sure I understand your point here?

Because otherwise we get failures about zero addresses (as the integration tests are expecting more to be setup):

```
ERC20HoldingDeposit: token cannot be 0x0
```

I need to spend some time updating the tests, however we need to get this merged so that the rest of the team can migrate over to Sepolia ASAP. The simplest solution was to use Goerli so that the tests would pass (for now).

File: script/utils/nfts/mintToken.ts (hunk 1 above)
Comment (ghost) [reply script/utils/nfts/mintToken.ts:32]: You don't need to supply the path to the ABI. The third argument passed to `requiredOption`  is the default path to use if you don't supply it.

**Review by utx0 — APPROVED** [review]
LGTM

**Pull request merged**